api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -3663,3 +3664,31 @@ message SessionRecordingAccess {
     (gogoproto.jsontag) = ""
   ];
 }
+
+// SSMRun is emitted after an SSM document completes execution.
```

Comment:
```suggestion
// SSMRun is emitted after an AWS SSM document completes execution.
```

Extra context to indicate that this is AWS-specific will help for those who don't know that SSM is an AWS feature.